File: DemoServer/AdditionalLanguages/python/advanced/createDatabase/createDatabase.py
from database import DocumentStoreHolder
from demo_example import Example

#region Usings
from ravendb import CreateDatabaseOperation
from ravendb.serverwide.database_record import DatabaseRecord
#endregion

class CreateDatabase(Example):
    def run(self, run_params=None):
        database_name = "someDb"

        #region Demo
        try:
            #region Step_1
            database_record = DatabaseRecord(database_name)
            create_database_operation = CreateDatabaseOperation(database_record)
            #endregion

            #region Step_2
            DocumentStoreHolder.store().maintenance.server.send(create_database_operation)
            #endregion

        except Exception as e:
            # Database already exists
            pass
        #endregion

File: DemoServer/AdditionalLanguages/php/advanced/createDatabase/CreateDatabase.php
<?php

namespace RavenDB\Demo\advanced\createDatabase;

//region Usings
use RavenDB\Exceptions\ConcurrencyException;
use RavenDB\ServerWide\DatabaseRecord;
use RavenDB\ServerWide\Operations\CreateDatabaseOperation;
//endregion

use RavenDB\Demo\common\DocumentStoreHolder;

class CreateDatabase
{
    public function __invoke()
    {
        $databaseName = "someDb";

        //region Demo
        try {
            //region Step_1
            $databaseRecord = new DatabaseRecord($databaseName);
            $createDatabaseOperation = new CreateDatabaseOperation($databaseRecord);
            //endregion

            //region Step_2
            DocumentStoreHolder::getStore()->maintenance()->server()->send($createDatabaseOperation);
            //endregion
            
        } catch (ConcurrencyException $e) {
            // Database already exists
        }
        //endregion
    }
}
